docs/literate/src/files/adding_new_equations_literate/cubic_conservation_law_literate.jl

```diff
@@ -83,18 +83,13 @@ sol = solve(ode, SSPRK43(),
             save_everystep=false, callback=callbacks, maxiters=1e5);
   
 ## Print the timer summary
-summary_callback()
-#src # TODO summary_callback() is printing summary automatically (even with ;)
+## summary_callback()
 
 # That's it, you ran your first simulation using your new equation with Trixi! Now,
 # we can plot the solution at the final time using Plots.jl.
 
 using Plots
 plot(sol)
-#src # TODO: In the static tutorial version the plot (1) is slighly different than the executed one (1b).
-#src # Storing the link for possible changes.
-#src # ![tutorial_adding_new_equations_plot1](https://user-images.githubusercontent.com/12693098/111651488-91122980-8806-11eb-848c-af09f3af234c.png)
-#src # ![tutorial_adding_new_equations_plot1b](https://user-images.githubusercontent.com/74359358/126663757-b8fb3ecb-72c8-4246-ab64-2bb4dbdf621a.png)
```

Comment:
I would suggest that we should just use the executed plot and delete the saved links.

Reply:
Sounds reasonable to me